index.html

```diff
@@ -451,19 +457,19 @@ <h2 class="bannerbox__title">Can I help some other way?</h2>
                                 <div class="help-block with-errors"></div>
                             </div>
                             <div class="form-group checkbox checkbox--multi">
-                                <p>I can</p>
-                                <label>
-                                    <input type="checkbox" name="Capability" value="build links"/> Help build links with organizations and communities
-                                </label>
+                                <p>How can you help?</p>
                                 <label>
-                                    <input type="checkbox" name="Capability" value="community organize"/> Organize my own community
+                                    <input type="checkbox" name="Capability" value="build links"/> Do community outreach / Organize my own community
                                 </label>
                                 <label>
-                                    <input type="checkbox" name="Capability" value="collaboration"/> Set up collaboration with my organization
+                                    <input type="checkbox" name="Capability" value="community organize"/> Help with media /social media outreach
```

Comment:
The `value` attribute on these checkboxes no longer aligns properly with the relevant text. Can you please ensure that these match up properly?

e.g. `value="community organize"` should match with `Do community outreach / Organize my own community`.